### PR TITLE
adds rake task that exports data as CSV to /tmp dir

### PR DESCRIPTION
**Pull request by sdellis**
@kevinreiss This works well for me locally, but it's my first rake task.  Can you take a look and merge if it's copacetic?

### CODE REVIEW

File: lib/tasks/export.rake [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+  desc "Export Alumni"
+  # usage:  rake export:alumni
+  task alumni: :environment do
+    require 'csv'
+
+      a = Alumn.all
+
+      CSV.open("/tmp/alumni.csv", "a+") do |csv|
+        csv << [ "id","box","lname","fname","year","graduate","photos","oversize" ]
+        a.each { |row| csv << [ row.id, row.lname, row.fname, row.box, row.year, row.graduate, row.photos, row.oversize ] }
```
Comment (jpstroop) [inline lib/tasks/export.rake]: Tiny thing, but I think `Alum.find_each` (See [#find_each](http://api.rubyonrails.org/classes/ActiveRecord/Batches.html#method-i-find_each)) will be more efficient than `a = Alum.all` (make a bunch of `Alum` objects in memory, stored in `a`) followed by `a.each` to loop over them.

**Comment by sdellis** [comment]
Ok, that makes sense @jpstroop ... thanks!  I also realized that it's probably easiest to use a pipe delimiter instead of a comma since the data is loaded with commas.  I will touch this up before any merge...

**Comment by kevinreiss** [comment]
@sdellis can this be merged or are you still working on this? If you do not want to use pipe delimiters you can use the :quote_char param on a Ruby CSV object to set the enclosure string used for fields that may contain reserved characters. I think the CSV library will try and be smart and only use them when needed.

**Comment by sdellis** [comment]
@kevinreiss No, this is ready to be merged.  I will need to run the rake task on production to export the most recent data. 

FYI - There is a separate issue that I will be adding and working on today relating to the Historic Photos.  Mudd was not using the OIT database for updating Historic Photos data + there was an import issue where only half of the wrong data was loaded.

**Pull request merged**